database/migrations/2021_12_10_100000_create_membership_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class CreateMembershipTable extends Migration
+{
+/**
+* Run the migrations.
+*
+* @return void
+*/
+public function up()
+{
+DB::statement("
+CREATE TABLE `membership` (
+`id` BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+`name` VARCHAR (50) NOT NULL,
+`types` VARCHAR (50) NOT NULL,
+`duration` VARCHAR (50) NOT NULL,
+`price` FLOAT NOT NULL,
+`price_lei` FLOAT NOT NULL,
```

Comment:
I recommend to avoid this column. It is unfinished idea, that more complicates application than provides value.